open_schools_platform/parent_management/families/serializers.py

```diff
@@ -0,0 +1,8 @@
+import uuid
+
+from rest_framework import serializers
+
+
+class FamilySerializer(serializers.Serializer):
+    parent_profile = serializers.UUIDField(default=uuid.uuid4)
+    name = serializers.CharField(allow_blank=True, default=None, required=False, max_length=200)
```

Comment:
"allow_blank=True" means that family can have empty name. It is not ok. Delete this argument.